spec/test_app/app/routing/bulletins_routing_spec.rb

```diff
@@ -36,5 +36,15 @@
                                         format: 'json')
     end
   end
+
+  describe 'GET /v1/bulletins/sign' do
+    let(:sign) { get "/v1/bulletins/sign" }
+
+    it 'routes to bulletins#sign)' do
+      expect(sign).to route_to(controller: 'basechurch/v1/bulletins',
+                               action: 'sign',
+                               format: 'json')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.